ocaml/sdk-gen/go/gen_go_binding.ml

```diff
@@ -44,20 +44,84 @@ let render_api_messages_and_errors destdir =
   generate_file ~rendered:messages_rendered ~destdir
     ~output_file:"api_messages.go"
 
+let render_convert_header () =
+  let obj =
+    `O
```

Comment:
Some repetition here that could be reduced:
```
let name s = `O [("name", `String s); ("sname", `Null)] in
...
`A (List.map name ["fmt"; "math"; ..]
```

Reply:
These related code has been split to the following PR which will raised after this one got merged.